src/nodes/Image.js

```diff
@@ -53,7 +53,7 @@ const Image = TiptapImage.extend({
 						// only catch the drop if it contains files
 						if (event.dataTransfer.files && event.dataTransfer.files.length > 0) {
 							const coordinates = view.posAtCoords({ left: event.clientX, top: event.clientY })
-							const customEvent = new CustomEvent('image-drop', {
```

Comment:
Should we move this to a more specific plugin?

Reply:
I don't know because it still will be written with the image syntax in the markdown content.